b_lambda_layer_common/util/list_batcher.py

```diff
@@ -0,0 +1,36 @@
+import json
+import math
+from itertools import islice
+from typing import Any, Iterable, List
+
+
+class ListBatcher:
```

Comment:
Name it to ListBytesBatcher, because it feels like it is just a simple regular batcher.